src/languages/es.ts

```diff
@@ -2067,6 +2067,7 @@ export default {
         genericUpdateReportFieldFailureMessage: 'Error inesperado al actualizar el campo. Por favor, inténtalo más tarde',
         genericUpdateReporNameEditFailureMessage: 'Error inesperado al cambiar el nombre del informe. Vuelva a intentarlo más tarde.',
         noActivityYet: 'Sin actividad todavía',
+        visibilityPublicPrompt: 'Anyone can find this room',
```

Comment:
@tienifr Looks like you have missed to replace it with Spanish translation.

In fact we already have the text in newRoomPage.publicDescription. Should we reuse it instead?